packages/cli/lib/projects.js

```diff
@@ -405,13 +405,19 @@ const pollProjectBuildAndDeploy = async (
   buildId,
   silenceLogs = false
 ) => {
-  const buildStatus = await pollBuildStatus(
-    accountId,
-    projectConfig.name,
-    buildId,
-    null,
-    silenceLogs
-  );
+  let buildStatus;
+  try {
+    buildStatus = await pollBuildStatus(
+      accountId,
+      projectConfig.name,
+      buildId,
+      null,
+      silenceLogs
+    );
+  } catch (e) {
+    //TODO: Figure out how we want to actually handle this
```

Comment:
I'm going to echo Camden's suggestion for `logApiErrorInstance`. That function checks whether the error is returned from `axios`, and if it's not, defaults to `logErrorInstance`, which works with generic errors. It's served me well in migration work.